cpp11test/src/test-as.cpp

```diff
@@ -285,7 +285,7 @@ context("as_cpp-C++") {
     expect_true(x2[1] == 2.);
     expect_true(x2[2] == 3.);
 
-    auto x3 = cpp11::as_cpp<const cpp11::doubles&>(r);
+    auto x3 = cpp11::as_cpp<cpp11::decay_t<const cpp11::doubles&>>(r);
```

Comment:
I'd just delete these test cases; no human should write `as_cpp<non_decayed>`

Reply:
yeah, I guess there is little point in keeping them